### ISSUE TITLE
Quick Pick: screen reader does not read out the list of visible items

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/vscode#52383]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : HighSierra10.13.5 

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader: VoiceOver

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "User Snippets" from dropdown..

**Actual:**
VoiceOver does not read the list items of the treeview.(json snippets files)

**Expected:**
VoiceOver should read all the list items including which are new snippets and which are existing.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not get the information about the list items.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode UserSnippets VoiceOver not reading list items.pptx](https://github.com/Microsoft/vscode/files/2117958/A11y.VSCode.UserSnippets.VoiceOver.not.reading.list.items.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Assigning this to September since a lot of accessibility issues are tied to this one
@chrmarti if you do not have time for this feel free to move to October

**Comment by chrmarti** [comment]
Upstream issue, filed https://github.com/electron/electron/issues/14795. /cc @bpasero

**Comment by bpasero** [comment]
@chrmarti thanks. do not forget to add the "electron" label as well.

**Comment by bpasero** [comment]
@chrmarti @isidorn instead of waiting for the upstream fix which will probably take a long time (if any), can we not just use tree-aria roles for all lists if we detect that we run on macOS?

**Comment by bpasero** [comment]
Oh, it looks like it also does not work for the normal quick open which is using a tree :-/

**Comment by chrmarti** [comment]
@bpasero I have tried `tree`, `list` and `listbox`. None worked.

**Comment by chrmarti** [comment]
Fixed in Electron 4.

**Comment by bpasero** [comment]
We will ship with Electron 4 with our upcoming release `1.36` this week. As such I am closing this issue. To benefit from the update already today, consider to use our insiders version: https://code.visualstudio.com/insiders/
